MongoProject/src/main/java/com/example/demo/Controller/UserController.java

```diff
@@ -2,52 +2,81 @@
 
 import java.util.List;
 
-
+import org.slf4j.LoggerFactory;
+import org.slf4j.Logger;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
 
 import com.example.demo.model.User;
-
+import com.example.demo.service.IUserService;
 import com.example.demo.service.UserService;
 
+import io.prometheus.client.Counter;
+
+
+
 
 @RestController
+
 public class UserController {
+	
+	Logger log =LoggerFactory.getLogger(this.getClass());
+	
 	@Autowired
 	public UserService userService;
+	@Autowired
+	private IUserService userServ;
+	
+	static final Counter counter = Counter.build()
+			.name("total_request")
+			.help("Total number of requests")
+			.register();
+	
+	@GetMapping(value="/{pageNo}/{pageSize}")
+	public List<User> getPaginated(@PathVariable int pageNo,@PathVariable int pageSize){
+		return userServ.findPaginated(pageNo, pageSize);
+	}
 	
-	@RequestMapping("/create")
-	public String create(@RequestParam String fname,@RequestParam String lname,@RequestParam String uemail,@RequestParam long ucontact,@RequestParam long uid) 
+	@PostMapping(value="/create")

```

Comment:
REST structure isn't correct with standards.
As we are working on User as resource, REST api contracts should be
* create user POST --> /user
* update user PUT --> /user/{user_id}
* delete user DELETE --> /user/{user_id}
* List all GET --> /user?page=&size=
* Get one GET --> /user/{user_id}